### PR TITLE
nzbget fixed

### PR DESCRIPTION
**Pull request by ob0t**
Tweaked the nzbget plugin to download from the DNZB API for newzbin posts. Should fix the issues with newzbin for nzbget users. I guess a similar fix could be applied to the blackhole method..

### CODE REVIEW

File: app/lib/nzbget.py [hunk 1]
```diff
@@ -45,8 +45,22 @@ class nzbGet():
...
+            if nzb.source == 'newzbin':
+                newzbinurl = 'http://www.newzbin.com/api/dnzb/'
+                newzbinargs = { 'username' : newzbinuser,
+                                'password' : newzbinpass,
+                                'reportid' : str(nzb.id) }
+                log.info('Receiving report ' + str(nzb.id) + ' from newzbin')
+                newzbindata = urllib.urlencode(newzbinargs)
+                nzburl = urllib2.Request(newzbinurl, newzbindata)
+            else:
+                log.error('Downloading '+nzb.url)
```
Comment (khellang) [inline app/lib/nzbget.py:61]: Great addition! But you probably want to change this line before a merge ;)

Comment (ob0t) [inline app/lib/nzbget.py:61]: probably not the neatest way to do it, but not sure i see what you mean - i take username/pass from the config (stored as newzbinuser/pass via config.get above) and feed them to the API with the NZB ID as an HTTP PUSH. Did I miss something?

Comment (khellang) [inline app/lib/nzbget.py:61]: Hehe, just thinking of the log.error statement :) Don't think you should log every download that's not from newzbin as an error ;)

Comment (ob0t) [inline app/lib/nzbget.py:61]: hah. good point (although maybe the newzbin guys would disagree!)

Comment (RuudBurger) [inline app/lib/nzbget.py:61]: Used your code and made it more "general". Thanks ;)

**Pull request merged**